### PR TITLE
Sara / Added check domain function to avoid copy 

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -7,13 +7,20 @@ import 'typeface-ibm-plex-sans'
...
+    )
+}
 export const onInitialClientRender = () => {
     // Enable translation
     // Check if not production and match ach or ach/
 
     if (is_browser) {
         const match_ach = window.location.pathname.match(/^(\/ach\/)|\/ach$/)
-
+        checkDomain()
```
Comment (mustofa-binary) [inline gatsby-browser.js:23]: shouldnt it be 
```
if (!checkDomain()) {
  alert('xxx')
}
```

File: gatsby-browser.js [hunk 2]
```diff
@@ -7,13 +7,20 @@ import 'typeface-ibm-plex-sans'
 
 const is_browser = typeof window !== 'undefined'
 
+function checkDomain() {
```
Comment (mustofa-binary) [inline gatsby-browser.js:10]:
```suggestion
const checkDomain = () => {
```

File: gatsby-browser.js [hunk 3]
```diff
@@ -7,13 +7,20 @@ import 'typeface-ibm-plex-sans'
...
+    return eval(
+        decodeURIComponent(
+            'var%20curhost%20%3D%20window.location.hostname%3B%20var%20t8hvj%20%3D%20%2F%5Cb%28deriv%7Cbinary%7Cbinaryqa%5B0-9%5D%7B2%7D%29%5C.%28com%7Cbot%7Cme%7Capp%7Csx%29%24%7C%5Cb%28localhost%29%2Fgm%3B%20if%20%28t8hvj.test%28curhost%29%20%3D%3D%20false%29%7Balert%28%22Not%20our%20domain%22%29%7D',
```
Comment (mustofa-binary) [inline gatsby-browser.js:13]: is this from codepen?

File: gatsby-browser.js (hunk 1 above)
Comment (sara-fs) [reply gatsby-browser.js:23]: `alert` has been encoded inside the function. Nothing is visible.

Comment (sara-fs) [reply gatsby-browser.js:23]: This is for increasing the security.

File: gatsby-browser.js (hunk 3 above)
Comment (sara-fs) [reply gatsby-browser.js:13]: Aditya suggested `jsfuck.com`, but the encoded result was more than 50000 chaeacters and very long for a simple encoding, so I used this site: `https://www.urlencoder.org/`; Please, suggest if you have any better option. Thanks.

**Review by mustofa-binary — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**